librz/debug/p/native/bsd/bsd_debug.c

```diff
@@ -675,6 +696,51 @@ RzList *bsd_thread_list(RzDebug *dbg, int pid, RzList *list) {
 		rz_list_append(list, pid_info);
 	}
 
+	free(kp);
+	return list;
+#elif __NetBSD__
```

Comment:
Please move this block into a static function. It gets confusing with the `#ifdef`